lib/common/client/base.rb

```diff
@@ -119,10 +117,26 @@ def request(method, path, params = {}, headers = {}, options = {}) # rubocop:dis
                       else
                         Common::Client::Errors::ClientError
                       end
+        status, body = response_hash_from_error(e)[:response].try(:values_at, :status, :body)
+        raise error_class.new(e.message, status, body)
+      end
+
+      def response_hash_from_error(error)
+        return { response: nil } unless error.respond_to?(:response)
+
+        resp = error.response
+
+        if %i[status body headers].all? { |method| resp.respond_to? method }
+          return {
+            response: {
+              status: resp.status,
+              body: resp.body,
```

Comment:
What kind of data is in a `Timeout::Error` or  `Faraday::TimeoutError` response?
As Michael pointed out most external service calls go through this code. 
We don't want PII ending up in Sentry.

Reply:
>We don't want PII ending up in Sentry.

Good point! And not sure what those errors hold --I'd like to find out!